crypto/fipsmodule/sha/sha256.c

```diff
@@ -163,14 +198,41 @@ static int sha256_final_impl(uint8_t *out, size_t md_len, SHA256_CTX *c) {
   return 1;
 }
 
-int SHA256_Final(uint8_t out[SHA256_DIGEST_LENGTH], SHA256_CTX *c) {
+int SHA256_Final(uint8_t out[SHA256_CHAINING_LENGTH], SHA256_CTX *c) {
```

Comment:
Why is this change needed? If not needed, we prefer not to change the function signature between its declaration in sha.h and its definition here.

Reply:
Good catch! This is a mistake. This should not have been changed.